sched/Kconfig

```diff
@@ -622,6 +622,12 @@ config SCHED_THREAD_LOCAL
 		This option enables architecture-specific TLS support (__thread/thread_local keyword)
 		Note: Toolchain must be compiled with '--enable-tls' enabled
 
+config UID_DEFAULT
```

Comment:
Where is CONFIG_UID_DEFAULT used?

Reply:
used by builtin apps side if not configured.
https://github.com/apache/nuttx-apps/pull/1691

Reply:
> used by builtin apps side if not configured.

If it effects apps/ (which are optional), then it should not be in an OS Kconfig file.  It should be an an apps/ Kconfig file.

Reply:
I think this idea comes from Android where each application runs on its own UID? Do I get the idea right?

What if we set `UID_DEFAULT` to `0` (`root`)?

Reply:
> I think this idea comes from Android where each application runs on its own UID? Do I get the idea right?

Yes, we use UID to isolate or share some user data, permission control in some complicated scenes.  But it is not being developed in Nuttx kernel.